core/sr-primitives/src/generic/digest.rs

```diff
@@ -69,8 +71,11 @@ pub enum DigestItem<Hash, AuthorityId, SealSignature> {
 	/// block. It is created for every block iff runtime supports changes
 	/// trie creation.
 	ChangesTrieRoot(Hash),
-	/// Put a Seal on it
+	/// The old way to put a Seal on it.  Deprecated.
+	#[deprecated]
 	Seal(u64, SealSignature),
+	/// Put a Seal on it
+	Consensus(ConsensusEngineId, Vec<u8>),
```

Comment:
I wonder if we should introduce `Seal2` and `ConsensusSignal` with the same definition but different semantic meanings. `Seal2` is for seals (applied by consensus engine after the runtime execution) and `ConsensusSignal` is for messages coming from the chain _to_ the consensus engine (e.g. GRANDPA set change signal).

Reply:
Maybe.  Not sure it should block merge, though.

Reply:
@rphmeier do you anticipate scenarios where we need to distinguish between them without actually knowing what the messages are?